models/VGGmod.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+import torch.nn as nn
+from torchvision import models
+
+
+class VGGmod(nn.Module):
+
+    def __init__(self, num_classes=1000, init_weights=True):
+        super().__init__()
+
+        self.num_classes = num_classes
+
+        self.features = models.vgg11(pretrained=True).features
+
+    def forward(self, x):
+        self.activs = []
+        for layer in self.features:
```

Comment:
Do we want to save the results after the forward pass through each module (e.g., conv, relu, pooling,...), or just the activations after each relu activation step?

Reply:
Isolating them to store just some of the layers would be more work, and we might as well have all of them in case we decide they could be useful for something during analysis.